src/components/SearchOptions/SearchOptions.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './SearchOptions.css'
+import SolidButton from '../../components/SolidButton/SolidButton';
+const SearchOptions = (props) => {
+    if (props.SearchState === true) {
+        return (
+
+            <div className="result-selection" >
+                <SolidButton
+                    className="result-nobackground-button"
```

Comment:
Since these two cases return almost identical things except for the classNames, instead you can consolidate the code to something like this: 

```
const SearchOptions = (props) => {
    return (
           <div className='result-selection'>
               <SolidButton className={`results-${props.SearchState === true && 'nobackground-'}button`}>
```
note that `results-${props.SearchState === true && 'nobackground-'}button` is surrounded by backticks!
you can apply the same logic to other places in this file.